tests/test262-es6-excludelist.xml

```diff
@@ -86,6 +86,7 @@
   <test id="built-ins/RegExp/prototype/Symbol.replace/y-set-lastindex.js"><reason></reason></test>
   <test id="built-ins/RegExp/prototype/Symbol.search/get-sticky-coerce.js"><reason></reason></test>
   <test id="built-ins/RegExp/prototype/Symbol.search/get-sticky-err.js"><reason></reason></test>
+  <test id="built-ins/RegExp/prototype/Symbol.split/coerce-limit.js"><reason></reason></test>
```

Comment:
Please add a short comment to reason block, for example: won't fix due to spec change - #3942